frame/staking/src/pallet/mod.rs

```diff
@@ -173,6 +173,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxUnlockingChunks: Get<u32>;
 
+		/// A hook called when any staker is slashed. Mostly likely this can be a no-op unless
+		/// there are nomination pools.
```

Comment:
```suggestion
		/// A hook called when any staker is slashed. Mostly likely this can be a no-op unless 
		/// if other pallets exist that are affected by slashing per-staker.
```